src/sentry/snuba/metrics/extraction.py

```diff
@@ -85,6 +85,8 @@
     "<=": "lte",
     ">": "gt",
     ">=": "gte",
+    "IN": "eq",
```

Comment:
Why not have an `"in"` CompareOp? Re-using "eq" seems confusing.

Reply:
I agree it can seem confusing but all of the examples of relay metric config that perform IN or NOT IN are using `"op": "eq"` instead of "op": "IN". My guess is that `in` is not even a valid operator.